.github/workflows/release-workflow.yaml

```diff
@@ -1,34 +1,28 @@
 name: Release new version
 
 on:
-  workflow_call:
+  workflow_dispatch:
     inputs:
       majorRelease:
         required: false
-        type: string
         description: 'Release as version. Example values are: minor, major, 5.6.1, 1.0.0. Leave empty for regular release.'
-      branch:
-        required: true
-        type: string
-        description: 'main or 0.x branch'
       tag:
         required: false
-        type: string
         description: 'Prerelease tag name. Leave empty for regular release.'
 
 jobs:
-  # integration-tests:
-  #   uses: ./.github/workflows/integration-tests.yaml
+  integration-tests:
+    uses: ./.github/workflows/integration-tests.yaml
 
   publish:
     runs-on: ubuntu-latest
-    # needs: integration-tests
+    needs: integration-tests
     steps:    
     - name: Check out code
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
-        ref: ${{ inputs.branch }}
+        ref: main
```

Comment:
let's remove this line, in best case it's redundant but if we were to try to release from a branch (e.g. in case of an urgent fix) this would release main instead.